src/components/workflowEditor/Workflow.tsx

```diff
@@ -203,47 +225,57 @@ export class Workflow extends Component<WorkflowProps, WorkflowState> {
                 sourceType={node.sourceType}
                 regex={node.regex}
                 primaryBranchAfterRegex={node.primaryBranchAfterRegex}
+                to={this.openCIPipeline(ci)} //ci attribites for a git material
+                handleGoToWorkFlowEditor={(e) => {
```

Comment:
check & update as discussed

Reply:
have updated handleGoToWorkflowEditor in props of Static Node - (e?: any) => void, hence done